lang/wasm_alien_thread_runner.cc

```diff
@@ -38,6 +42,13 @@ alien_thread_runner::alien_thread_runner()
         sigfillset(&mask);
         auto r = ::pthread_sigmask(SIG_BLOCK, &mask, nullptr);
         throw_pthread_error(r);
+
+        errno = 0;
+        int nice_value = nice(10);
+        if (nice_value == -1 && errno != 0) {
+            wasm_logger.warn("Unable to renice the alien thread; the thread will compete with reactor. Try adding CAP_SYS_NICE");
```

Comment:
might want to print out `errno` as part of the logging message, to assist the sysadm to identify the root cause of the failure?